src/main/java/dev/j3fftw/litexpansion/items/CargoConfigurator.java

```diff
@@ -53,8 +53,7 @@ public void onCargoConfiguratorItemClick(PlayerInteractEvent e) {
         if (e.getItem() == null || e.getMaterial() != Material.COMPASS) return;
 
         final ItemStack clickedItem = e.getItem();
-        final SlimefunItem item = SlimefunItem.getByItem(e.getItem());
-        if (item == null || !item.getID().equals(Items.CARGO_CONFIGURATOR.getItemId())) return;
+        if (!this.isItem(clickedItem)) return;
```

Comment:
mind explaining this change?
also shouldnt the return be if code {
return; }

Reply:
> also shouldnt the return be if code {
return; }

I would prefer this too, but LX uses the one-line guard block pattern everywhere. It used to be that too.

> mind explaining this change?
Previously it checked if the currently held item is either null or not the cargo configurator.
.isItem() basically does the same just better performance-wise and id-independent.
Because we are currently in the Cargo Configurator class, we can reference this here and save ourselves any item comparisons, simply check if the item is this exact item.

Reply:
> I would prefer this too, but LX uses the one-line guard block pattern everywhere. It used to be that too.

im fine changing this since i believe having no bodies on an if is a bad practice

Reply:
Then it should be changed everywhere though, which should probably just be done all at once later down the line.
And I think it is a bad practice too as it is not immediately clear where things start or end.

Reply:
Okay im fine keeping it like this then will change it my self later on